### ISSUE TITLE
Auto-scroll mode for reading?  

### ISSUE THREAD

**Issue by nmukh** [johnfactotum/foliate#507]
Foliate is beautiful software. Would it be possible to implement auto-scroll, perhaps in scrolled or continuous mode, with adjustable scrolling speed? I find hands-free reading very useful.

**Comment by digitalethics** [comment]
I think the auto-scroll mode in Evince works very well. Okular is struggling with implementing this and it might be good to take a look at the [problems](https://invent.kde.org/graphics/okular/-/merge_requests/50) to avoid them.

**Comment by nmukh** [comment]
Evince and Okular both have auto-scroll, but Evince does not support epub and Okular supports epub but does not reflow text. Foliate has been ideal because it allows annotations (a feature Calibre, my usual epub reader, lacks). Implementing auto-scroll would make it ideal.

**Comment by digitalethics** [comment]
> Implementing auto-scroll would make it ideal.

Yes, I agree. I didn't mean to refer to the others as alternatives to Foliate but as good examples to learn from so that implementation strategies can be discussed for Foliate.
